integration_test/third_party_apps_data/applications/hbase/centos_rhel/install

```diff
@@ -1,11 +1,13 @@
 set -e
 
+HBASE_VERSION=$(curl https://apache.mirror.gtcomm.net/hbase/stable/ | grep --extended-regexp 'hbase-[0-9.]+-bin.tar.gz' | grep --extended-regexp --only-matching '[0-9]+\.[0-9]+\.[0-9]+' | head -1)
+
 sudo yum -y install \
     java-1.8.0-openjdk java-1.8.0-openjdk-devel wget
 
 sudo mkdir /opt/hbase
-wget http://apache.mirror.gtcomm.net/hbase/2.4.9/hbase-2.4.9-bin.tar.gz
-sudo tar -xvf hbase-2.4.9-bin.tar.gz -C /opt/hbase --strip 1
+wget http://apache.mirror.gtcomm.net/hbase/$HBASE_VERSION/hbase-$HBASE_VERSION-bin.tar.gz
```

Comment:
FYI:

The google bash style guide says to always double quote variables: `"$HBASE_VERSION"`. IIUC this is to handle variable values that contain spaces "as you expect" instead of "how bash is implemented". There is also a linter for this in google3.

Since this is not in google3 this is optional, but IMO it's a good habit.